revolv/base/management/commands/seed.py

```diff
@@ -119,7 +120,8 @@ class ProjectSeedSpec(SeedSpec):
         "location": "1238 5th Street, Berkeley, CA, United States",
         "location_latitude": 37.87968940000000,
         "location_longitude": -122.30289330000000,
-        "end_date": datetime.date(2175, 1, 1),
+        "start_date": datetime.date(2014, 1, 1),
+        "end_date": datetime.date(2015, 1, 1),
```

Comment:
This one is important to keep as it is, since the project is supposed to be completed, and the date should thus be in the past.